GliaWidgets/Sources/CallVisualizer/CallVisualizer.swift

```diff
@@ -78,6 +80,8 @@ public final class CallVisualizer {
     /// - Parameter source: The current viewController to present from.
     ///
     public func showVisitorCodeViewController(from source: UIViewController) {
+        environment.log.prefixed(Self.self).info("Show visitor code")
+        environment.log.prefixed(Self.self).info("Show Visitor Code Dialog")
```

Comment:
do we need to have this duplication?

Reply:
Log on line 83, seems to be not present in Android, so it should be removed here as well. It is present in comparison table though 03ec82f55b8134c57ab9d2d986b5a986b84fdb08.

Reply:
Sorry, there is no "Show visitor code" on Android. Most probably, indeed, to avoid duplication. I just missed to remove this event from the spreadsheet.

So you can remove "Show visitor code".